common/util/init_command_line.cc

```diff
@@ -19,11 +19,26 @@
 #include "absl/flags/flag.h"
 #include "absl/flags/parse.h"
 #include "absl/flags/usage.h"
+#include "absl/flags/usage_config.h"
+#include "common/util/verible_build_version.h"
 
 namespace verible {
 
+std::string GetBuildVersion() {
```

Comment:
`static ... GetBuildVersion()` if this is not intended for use outside this translation unit

Reply:
yes, static it should be.

Reply:
done.